lib/views/item/item-detail.js

```diff
@@ -180,12 +184,60 @@ class ItemDetail extends Component {
       <View style={styles.distance}>
         <Image
           style={styles.distanceImage}
-          source={require('../../images/item-scene-room.png')}/>
+          source={{
+            uri: 'ic_place',
+          }}/>
         <Text style={styles.distanceLabel}>{label}</Text>
       </View>
     );
   }
 
+  renderMap(item = {}) {
+    const beaconLocation = item.location;
+
+    if (!beaconLocation ||
+      typeof beaconLocation.latitude !== 'number' ||
+      typeof beaconLocation.longitude !== 'number') {
+      return null;
+    }
+
+    return (
+      <View
+        style={styles.mapContainer}>
+        <MapView
+          style={StyleSheet.absoluteFillObject}
+          initialCenterCoordinate={{
+            latitude: beaconLocation.latitude,
+            longitude: beaconLocation.longitude,
+          }}
+          annotations={[{
+            coordinates: [
+              beaconLocation.latitude,
+              beaconLocation.longitude,
+            ],
+            type: 'point',
+            id: 'marker1',
+            annotationImage: {
+              source: {
+                uri: 'ic_place',
```

Comment:
Again can we use `require()` to keep all our front-end image assets in one place?